src/Notepads/Controls/TextEditor/TextEditor.xaml.cs

```diff
@@ -889,5 +952,47 @@ private void GoToControl_OnDismissKeyDown(object sender, RoutedEventArgs e)
             GoToPlaceholder.Dismiss();
             TextEditorCore.Focus(FocusState.Programmatic);
         }
+
+        private void DrawLineHighlighter()
+        {
+            TextEditorCore.Document.Selection.GetRect(Windows.UI.Text.PointOptions.ClientCoordinates, out Windows.Foundation.Rect highlightRect, out var _);
+            LineHighlighter.Height = 1.35 * TextEditorCore.FontSize;
```

Comment:
How did you come up with this magic number? (1.35)

Reply:
On trying with different font sizes and different font factors, this is the one that worked perfectly.

Reply:
That explains. I have few more suggestions regarding to your PR and there are some places need to be polished. I can merge your PR right away and do it on my own or I can leave you comments. Which way do you prefer?

Reply:
Leave me comments and I will do it right away.

Reply:
1. Bottom edge of the highlighter is way too close to the bottom of the line. Increase it a little bit would make it better.
![image](https://user-images.githubusercontent.com/1166162/74585982-94017180-4f97-11ea-8cee-8b4d284a7758.png)

2. I do not think adding "Wrap Word" here is necessary since it is already accessible in right click context menu. As well for "Highlight Current Line", it is not something that user will change on a daily basis. So keeping it in the settings is good enough. Basically you can remove this menu completely and whenever user click on line number indicator, the GoTo should pop up.
![image](https://user-images.githubusercontent.com/1166162/74585990-af6c7c80-4f97-11ea-8f6d-bef6fb1dde30.png)

3. Remove the hint "Restore Default Zoom (Ctrl+0)" and remove the gap between four edges:
![image](https://user-images.githubusercontent.com/1166162/74586040-3de0fe00-4f98-11ea-8684-c947c08dc2dd.png)

4. Add a GoTo Text string right before the text box to make it clear. Otherwise user will be confused and will think it is the normal search bar:
![image](https://user-images.githubusercontent.com/1166162/74586051-6a951580-4f98-11ea-93dd-934262b489f9.png)

5. Make this feature enabled by default since there is no harm of doing so and most of the users won't know it if they do not go to settings on a daily basis:
![image](https://user-images.githubusercontent.com/1166162/74586083-bb0c7300-4f98-11ea-9d56-919821ba0162.png)

6. Increase the top and bottom gap of the search engine custom textbox will make it look better:
![image](https://user-images.githubusercontent.com/1166162/74586096-e2fbd680-4f98-11ea-88c8-c6fc8b7b27de.png)

Reply:
About changing the linehighlighter height, I chose 1.35 cause the selection height is also the same. 
![ApplicationFrameHost_2020-02-15_18-08-01](https://user-images.githubusercontent.com/51203900/74587912-37956500-501e-11ea-9b2a-d15f6611639f.png)
For my research into other text editors that implement the same behaviour, the bottom of the line is also a lot closer. I have no problem increasing the height but it looks weird when words are selected. 
![ApplicationFrameHost_2020-02-15_18-05-20](https://user-images.githubusercontent.com/51203900/74587913-39f7bf00-501e-11ea-977e-189524bfecae.png)
Shall I disable highlight when selection length is greater than zero?

Reply:
Ok, makes sense. Let's keep what it is. We can keep the higlight even if selection length is greater than zero.